tests/Models/Multiplayer/RoomTest.php

```diff
@@ -70,17 +71,50 @@ public function testMaxAttemptsReached()
     {
         $user = factory(User::class)->create();
         $room = factory(Room::class)->create(['max_attempts' => 2]);
-        $playlistItem = factory(PlaylistItem::class)->create([
+        $playlistItem1 = factory(PlaylistItem::class)->create([
+            'room_id' => $room->getKey(),
+        ]);
+        $playlistItem2 = factory(PlaylistItem::class)->create([
             'room_id' => $room->getKey(),
         ]);
 
-        $room->startPlay($user, $playlistItem);
+        $room->startPlay($user, $playlistItem1);
         $this->assertTrue(true);
 
-        $room->startPlay($user, $playlistItem);
+        $room->startPlay($user, $playlistItem2);
         $this->assertTrue(true);
 
         $this->expectException(InvariantException::class);
-        $room->startPlay($user, $playlistItem);
+        $room->startPlay($user, $playlistItem1);
+    }
+
+    public function testMaxAttemptsForItemReached()
```

Comment:
This looks like it's just testing the same thing twice, the previous test checks that the room limit is hit, this one should probably be checking the playlist limit is hit but not the room limit, so maybe it should be testing is other plays can still be started?

Reply:
it's already testing whether other item can still be started, just no assertion (now added).